rpc/client/evidence_test.go

```diff
@@ -113,6 +115,7 @@ func makeEvidences(
 }
 
 func TestBroadcastEvidence_DuplicateVoteEvidence(t *testing.T) {
+	time.Sleep(100 * time.Millisecond)
```

Comment:
Please add a comment on why this `Sleep` is necessary (and on what can be improved to remove this).

Reply:
Sorry for the ambiguity. My intention was to ask under what circumstances this `Sleep()` could be removed.